src/atoms/Icon/Icon.native.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import icons from '../../icons';
+
+const iconSize = {
+  s: 10,
+  m: 16,
+  l: 20,
+};
+
+const Icon = ({ size, name, fill }) => {
+  const IconComponent = icons[name];
+  return <IconComponent fill={fill} height={iconSize[size]} width={iconSize[size]} />;
```

Comment:
Should we expose other SVG style properties like `strokeWidth`, `fillOpacity?

Reply:
I'm thinking we should do `{...props}` so that we don't need to manually add support for all svg props